### ISSUE TITLE
Transition item is not translated using the admin translation domain

### ISSUE THREAD

**Issue by gremo** [yokai-php/sonata-workflow#7]
Here: https://github.com/yokai-php/sonata-workflow/blob/master/src/Admin/Extension/WorkflowExtension.php#L181 you are using:

```php
$menu->addChild(
    $admin->getLabelTranslatorStrategy()->getLabel($transition->getName(), 'workflow'),
    $options
);
```

Which is correct, because the extension should use the same label translation strategy of the admin. But translation domain should be the same of the admin, this could be easily fixed:

```php
$menu->setExtra('translation_domain', $admin->getTranslationDomain());
```

**Comment by gremo** [comment]
@yann-eugone thanks so much, yes, I'm using this bundle for a new project and I've found very useful. I have another idea, but I'll open another issue maybe (enhancement).

**Comment by yann-eugone** [comment]
Fixed as of [v0.2.0](https://github.com/yokai-php/sonata-workflow/releases/tag/v0.2.0)
